tests/protractor/e2e/forms/submit-delivery-form.specs.js

```diff
@@ -1,20 +1,74 @@
 const helper = require('../../helper'),
-  deliveryReport = require('../../page-objects/forms/delivery-report.po.js');
-
+  deliveryReport = require('../../page-objects/forms/delivery-report.po.js'),
+  common = require('../../page-objects/common/common.po.js');
+  
 describe('Submit Delivery Report', () => {
 
+  const contactId = '3b3d50d275280d2568cd36281d00348b';
+  const docs = [
+    {
+      _id: 'c49385b3594af7025ef097114104ef48',
+      reported_date: 1469578114543,
+      notes: '',
+      contact: {
+        _id: contactId,
+        name: 'Jack',
+        date_of_birth: '',
+        phone: '+64274444444',
+        alternate_phone: '',
+        notes: '',
+        type: 'person',
+        reported_date: 1478469976421
+      },
+      name: 'Number three district',
+      external_id: '',
+      type: 'district_hospital'
+    },
+    {
+      _id: contactId,
+      name: 'Jack',
+      date_of_birth: '',
+      phone: '+64274444444',
+      alternate_phone: '',
+      notes: '',
+      type: 'person',
+      reported_date: 1478469976421,
+      parent: {
+        _id: 'c49385b3594af7025ef097114104ef48',
+        reported_date: 1469578114543,
+        notes: '',
+        contact: {
+          _id: contactId,
+          name: 'Jack',
+          date_of_birth: '',
+          phone: '+64274444444',
+          alternate_phone: '',
+          notes: '',
+          type: 'person',
+          reported_date: 1478469976421
+        },
+        name: 'Number three district',
+        external_id: '',
+        type: 'district_hospital'
+      }
+    }
+  ];
+
+  const noteToCHW = 'Good news, Jack! Jack () has delivered at the health facility. We will alert you when it is time to refer them for PNC. Please monitor them for danger signs. Thank you!';
```

Comment:
Apparently constants are normally all-caps but we didn't follow this standard/convention all over our test code. Should we @garethbowen ?